modules/enum_account/main.py

```diff
@@ -95,8 +95,6 @@ def main(args, pacu_main):
 
     session.update(pacu_main.database, Account=account_data)
 
-    print('{} completed.\n'.format(module_info['name']))
```

Comment:
Took out the line because Ben made a comment that the Module Summary by itself was concise enough for the whole module. I can add it back in though. I added the print statement because it explicitly lets the user know the account that is being enumerated against.